charts/govuk-rails-app/templates/ingress.yaml

```diff
@@ -10,28 +10,27 @@ metadata:
     app.kubernetes.io/component: web
   annotations:
     {{- toYaml .Values.ingress.annotations | nindent 4 }}
+    external-dns.alpha.kubernetes.io/ingress-hostname-source: annotation-only
+    external-dns.alpha.kubernetes.io/hostname: {{ join "," .Values.ingress.hosts }}
 spec:
-  {{- if .Values.ingress.tls }}
+  {{- if .Values.ingress.hosts }}
   tls:
-    {{- range .Values.ingress.tls }}
     - hosts:
```

Comment:
I'd much prefer we just have it accept any hostname. It's easier to understand and easier to test, which will means fewer things to go wrong when we cut over in production.